src/settings.cpp

```diff
@@ -140,6 +142,9 @@ Settings::Settings(Root* root)
         root->monitors()->lock_number_changed();
     });
     g_settings = this;
+    for (auto i : attributes()) {
+        i.second->setWriteable();
```

Comment:
Why do we need this now and not before? Isn't this redundant with the individual setWriteable() statements above?

Reply:
This didn't cover all settings. And instead of explicitly making the remaining ≈5 settings I've decided to simply mark all settings writable, because all settings must be writable. We can omit the explicit `setWritable()` calls.

Reply:
I've removed the earlier setWritable() calls.